### PR TITLE
Better table view

### PR DESCRIPTION
**Pull request by RobbiNespu**
Current CSS table are not nice when td have longer value. It cramp up. Sending my patch for you review

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -291,3 +291,19 @@ content: 'Markdown';
...
+
+/* table */
+table {
+  border-spacing: 0;
+  border-collapse: collapse;
+}
+
+table th{
+  padding: 6px 13px;
+  border: 1px solid #dfe2e5;
```
Comment (athul) [inline assets/css/main.css:303]:
```suggestion
  border: 1px solid #dfe2e5;
  font-size: large;
```

Comment (athul) [reply assets/css/main.css:303]: We could maybe Increase the Font Size of the Table Header for a bit of emaphasis? What's your takeon it?

Comment (RobbiNespu) [reply assets/css/main.css:303]: what size suitable for this?

**Comment by athul** [comment]
Cool, Merging this 🎉 . Thanks for the PRs 🤗

**Review by athul — APPROVED** [review]

**Pull request merged**